src/search.c

```diff
@@ -499,6 +578,26 @@ static void do_search(struct manifest *MoM, char search_type, char *search_term)
 			continue;
 		}
 
+		/* record contentsize and includes for install size calculation */
+		struct bundle_result *bundle = NULL;
+		bundle = calloc(sizeof(struct bundle_result), 1);
```

Comment:
Optional: allocate all the bundle_results here (either pick filecount or count the submans), and pass them down (instead of the man that was being passed).